### ISSUE TITLE
Add string slices

### ISSUE THREAD

**Issue by tomcl** [fsprojects/Fable#358]
### Description

```
let s: string = "cat and dog"
printfn "%s" s.[2..8]
```

should print:
```
t and d
```

Currently slices on string are not implemented. This is a very basic operation and while it can be implemented as a function that loses the special syntax.

selection on string is currently implemented, so this fills in a gap.

**Comment by alfonsogarciacaro** [comment]
This should be fixed with fable-compiler@0.5.5 :+1:

**Comment by tomcl** [comment]
Brilliant!
    
    
    On 15/08/2016 20:10, Alfonso
      Garcia-Caro wrote:
    
    
      
      This should be fixed with fable-compiler@0.5.5 👍
      —
        You are receiving this because you authored the thread.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
          
          
        
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/fsprojects/Fable","title":"fsprojects/Fable","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/fsprojects/Fable"}},"updates":{"snippets":[{"icon":"PERSON","message":"@alfonsogarciacaro in #358: This should be fixed with fable-compiler@0.5.5 :+1:"}],"action":{"name":"View Issue","url":"https://github.com/fsprojects/Fable/issues/358#issuecomment-239898385"}}}
